frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -5,269 +5,270 @@
+                    [matChipInputAddOnBlur]="addOnBlur"
+                    (matChipInputTokenEnd)="add($event)"
+                  />
+                </mat-chip-list>
+              </mat-form-field>
+              <div class="error-message" *ngIf="isValidationError">{{ message }}</div>
+              <div
+                class="btn btn-waves-effect waves-dark grad-btn grad-btn-dark fs-16"
+                [class.disabled]="isValidationError"
+                (click)="reflectChange()"
+              >
+                <span>Update</span>
+              </div>
+            </div>
           </div>
-        </div>
-        <!-- Start date -->
-        <span class="fs-16 fw-light">
-          <strong class="text-light-black">Starts on:</strong>
-          {{ challenge['start_date'] | date: 'medium' }} {{ challenge['start_zone'] }}
-        </span>
-        <span *ngIf="isChallengeHost" class="fs-16 fw-light">
-          <a class="pointer fs-16" (click)="challengeDateDialog()">
-            <i class="fa fa-pencil" aria-hidden="true"></i>
-          </a>
-        </span>
-        <br />
-        <!-- End date -->
-        <span class="fs-16 fw-light">
-          <strong class="text-light-black">Ends on:</strong>
-          {{ challenge['end_date'] | date: 'medium' }} {{ challenge['end_zone'] }}
-        </span>
-        <span *ngIf="isChallengeHost" class="fs-16 fw-light">
-          <a class="pointer fs-16" (click)="challengeDateDialog()">
-            <i class="fa fa-pencil" aria-hidden="true"></i>
-          </a>
-        </span>
-        <br />
-        <span class="fs-16 fw-light" *ngIf="isChallengeHost">
-            <a class="pointer" (click)="togglePublishChallengeState()">
-              <strong class="text-light-black">{{ publishChallenge.state }} </strong>
-              <i class="{{ publishChallenge.icon }}" aria-hidden="true"></i>
-            </a>
-        </span>
-        <br />
-        <mat-checkbox
-          color="primary"
-          class="fw-semibold"
-          *ngIf="isChallengeHost"
-          [checked]="!challenge['is_registration_open']"
-          (click)="stopParticipation($event)"
-          ><strong class="text-light-black fs-15">Close Participation</strong>
-        </mat-checkbox>
-      </div>
-    </div>
-  </div>
+        </ng-template>
+      </mat-tab>
+
+      <!-- Edit Phase Details -->
+      <mat-tab label="Edit phase details"> 
+        <div class="row settings-section">
+          <div class="ev-action">
+            <div class="col-sm-6 col-xs-12 col-lg-6 col-lr-pad phase-select-box" *ngIf="isChallengeHost">
+              <div class="row margin-bottom-cancel phase-card">
+                <app-selectphase
+                  [phases]="filteredPhases"
+                  [phaseSelectionListType]="phaseSelectionListType"
+                  [phaseSelectionType]="phaseSelectionType"
+                  [phaseSelected]="phaseSelected()"
+                  #phaseselect
+                ></app-selectphase>
```

Comment:
One of the phases should be preselected on this tab. This would give hosts clear idea of what they can edit.